aiconfig-docs/docs/editor.md

```diff
@@ -0,0 +1,166 @@
+
+Click ▶️ at the bottom right of the cell to execute the prompt and see the model response. Support for streaming responses is coming soon!
+
+![image3](https://github.com/lastmile-ai/aiconfig/assets/129882602/df2718ba-41e8-46ac-88f6-0ab312ecdc6b)
+
+The outputs are saved to the AIConfig file by default.
+
+## Chain Prompts {#chain-prompts}
+
+You can chain your prompts via the cell reference names and handlebars syntax. For example, you can have a cell that uses GPT-4 to generate a haiku, and a GPT-3.5 cell that translates the message into a different language.
+
+![image7](https://github.com/lastmile-ai/aiconfig/assets/129882602/22243c9e-f615-4abd-9b59-7d9f9a54bb2f)
+
+## Create Prompt Templates {#prompt-templates}
+
+Prompt templates allow you to scale your prompts to different data inputs without needing to constantly modify the prompt itself. To do this in AIConfig Editor, variables are used to pass in data to prompts. You can set both global variables and local variables. Global Variables can be used across all prompts defined in the editor whereas Local Variables can only be used in the prompt cell they are defined for.
+
+**Global Variables**
+You can set global variables to be used across all cells in the editor. Click on `Global Variables` at the top of the editor to expand the form to enter your global variables.
+
+![image5](https://github.com/lastmile-ai/aiconfig/assets/129882602/9633b389-a9ae-4bbd-b9bd-5c965dbbdcaf)
+
+**Local Variables**
+You can set local variables to be used in specific cells in the editor. In the cell, expand the right pane and select `Local Variables (Parameters)`.
+
+:::note
+Local parameters will override the global parameters if they have the same name.
+:::
+
+![image6](https://github.com/lastmile-ai/aiconfig/assets/129882602/3c4408e4-be34-4b13-bddc-2dff5df88bcd)
+
+**Creating Prompt Templates**
+Prompt templates are created using [handlebars syntax](https://handlebarsjs.com/guide/) for the variables. Here is an example where `{{language}}` is defined as a global variable. You can easily change the values of the variable but keep the prompt template the same.
+
+![image4](https://github.com/lastmile-ai/aiconfig/assets/129882602/4333b532-bc04-41c4-adcb-ce1e9c8ef8ea)
+
+## More Resources
+
+Check out these resources on how you can use your AIConfig created from your AIConfig Editor in your application code.
+Getting Started Tutorial
+AIConfig API Reference
+Cookbooks
+
+## Coming Soon
+
+- **Streaming model responses in Editor.** Streaming is the printing or processing of the beginning of the completion before the full completion is finished.
+- **Support for non-default models in Editor.** AIConfig Editor currently supports the default models available with AIConfig - see here. We will soon be adding support for non-default models via model parser extensions.
```

Comment:
Also say you can see our `Gradio` cookbook to see how we imported our HuggingFace extension containing other cool model parsers like
- text-to-image
- text-to-audio
- image-to-text
- audio-to-text
- text-summarization 
- and much more!!!